### DISCUSSION TITLE
HOWTO store FHIR-resource object in a MongoDB

### DISCUSSION THREAD

**Discussion by Brilbroeder** [FirelyTeam/firely-net-sdk#2618] [Q&A]
MongoDb driver is complaining about not being configured to serialize Fhir poco's.

**Comment by ewoutkramer** [comment]
We are using MongoDb ourselves in our own products, and we are using the provided serializers in this SDK to serialize to json and then store them. The Poco's are complex (subclassing, etc) and never meant to be true DTOs, so I doubt any standard serializer or Mongo client will be able to process them out of the box!
